eap8/README.md

```diff
@@ -0,0 +1,14 @@
+= You can optionally get started using vagrant along with libvirt that would configure below settings.
```

Comment:
Rename this eap8 folder to something like eap8-vagrant and move it to the admin1 folder at the top-level of this repo